products/chromeos.md

```diff
@@ -0,0 +1,153 @@
+
+-   releaseCycle: "111"
+    releaseDate: 2023-03-01
+    eol: 2023-03-29
+
+-   releaseCycle: "110"
+    releaseDate: 2023-02-01
+    eol: 2023-03-01
+
+-   releaseCycle: "109"
+    releaseDate: 2023-01-10
+    eol: 2023-02-01
+
+-   releaseCycle: "108"
+    releaseDate: 2022-11-29
+    lts: true
+    eol: 2023-03-01
+
+-   releaseCycle: "107"
+    releaseDate: 2022-10-25
+    eol: 2022-11-29
+
+-   releaseCycle: "106"
+    releaseDate: 2022-09-27
+    eol: 2022-10-25
+
+-   releaseCycle: "105"
+    releaseDate: 2022-08-30
+    eol: 2022-09-27
+
+-   releaseCycle: "104"
+    releaseDate: 2022-08-02
+    eol: 2022-08-30
+
+-   releaseCycle: "103"
+    releaseDate: 2022-06-21
+    eol: 2022-08-02
+
+-   releaseCycle: "102"
+    releaseDate: 2022-05-26
+    lts: true
+    eol: 2022-11-29 
+
+-   releaseCycle: "101"
+    releaseDate: 2022-04-26
+    eol: 2022-05-26
+
+-   releaseCycle: "100"
+    releaseDate: 2022-03-29
+    eol: 2022-04-26
+
+-   releaseCycle: "99"
+    releaseDate: 2022-03-01
+    eol: 2022-03-29
+
+-   releaseCycle: "98"
+    releaseDate: 2022-02-02
+    eol: 2022-03-01
+
+-   releaseCycle: "97"
+    releaseDate: 2022-01-04
+    eol: 2022-02-02
+
+-   releaseCycle: "96"
+    releaseDate: 2021-12-06
+    eol: 2022-01-04
+
+-   releaseCycle: "95"
+    releaseDate: 2021-10-19
+    eol: 2021-12-06
+
+-   releaseCycle: "94"
+    releaseDate: 2021-09-21
+    eol: 2021-12-06
+
+-   releaseCycle: "93"
+    releaseDate: 2021-08-31
+    eol: 2021-09-21
+
+-   releaseCycle: "92"
+    releaseDate: 2021-07-20
+    eol: 2021-08-31
+
+-   releaseCycle: "91"
+    releaseDate: 2021-05-21
+    eol: 2021-07-20
+
+-   releaseCycle: "90"
+    releaseDate: 2021-04-14
+    eol: 2021-05-21
+
+-   releaseCycle: "89"
+    releaseDate: 2021-03-09
+    eol: 2021-04-14
+
+-   releaseCycle: "88"
+    releaseDate: 2021-01-19
+    eol: 2021-03-09
+
+-   releaseCycle: "87"
+    releaseDate: 2020-11-17
+    eol: 2021-01-19
+
+-   releaseCycle: "86"
+    releaseDate: 2020-10-06
+    eol: 2020-11-17
+
+-   releaseCycle: "85"
+    releaseDate: 2020-08-25
+    eol: 2020-10-06
+
+---
+> [chromeOS](https://www.google.com/chromebook/chrome-os/) is a Linux-based operating system developed and designed by Google. It is derived from the open-source ChromiumOS and uses the Google Chrome web browser as its principal user interface.
+
+Chrome OS follows a regular release schedule, similar to other operating systems. New versions of Chrome OS are released approximately every six weeks. These releases include updates, bug fixes, security enhancements, and sometimes new features. Chrome OS releases are identified by version numbers, such as "Chrome OS 95," "Chrome OS 96," and so on.
+
+Devices receive automatic software updates for approximately 6.5 years from their original launch date. During the first 5 years, devices receive both security and feature updates. In the last 1.5 years, devices continue to receive security updates but might not receive feature updates.
```

Comment:
This needs to be sourced. Where can I confirmed this (would prefer an official "Google" link if possible).